plugins/modules/infinity.py

```diff
@@ -110,26 +110,38 @@
   type: str
   sample: '1501'
 ip_info:
-  description: When reserve next available IP address from a network, the IP address info is returned.
+  description:
+    - When reserve next available IP address from a network, the IP address info is returned.
+    - Please note that the value is a B(string) containing JSON data.
   returned: success
   type: str
-  sample: '{"address": "192.168.10.3", "hostname": "", "FQDN": "", "domainname": "", "id": 3229}'
+  sample: >-
+    {
+      "address": "192.168.10.3",
+      "hostname": "",
+      "FQDN": "",
+      "domainname": "",
+      "id": 3229
+    }
```

Comment:
While this is more readable, this is also a different value: there's now a space after `{` and before `}`. I guess that's OK though.